tf-build-tools/tf-maven-plugin/src/main/java/com/deliveredtechnologies/maven/terraform/TerraformPackage.java

```diff
@@ -50,6 +55,7 @@
     tfModulesDir,
     tfRootDir,
     tfVarFiles,
+    tfVars,
```

Comment:
Remove TODO above, pls. TODOs are not for notes, they are for things that actually need to be done.